spec/controllers/v1/sessions_controller_spec.rb

```diff
@@ -86,6 +86,7 @@ def expect_logger_msg(level, msg)
                 .to trigger_statsd_increment(described_class::STATSD_SSO_NEW_KEY, tags: ["context:#{type}"], **once)
 
               expect(response).to have_http_status(:found)
+              # FIXME: should this not be a PINT url?
```

Comment:
@patrickvinograd / @hinzed1127 I believe these response.location values should be pointing to PINT urls.  I'm guessing that its just using the same mocked build response values that v0 is using.  However nothing jumped out at me in the `spec/support/saml/response_builder.rb` module.  Any ideas to where this is being set?

Reply:
think I figured it out, working on the changes now...https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/factories/rubysaml_settings.rb